src/flux/media.coffee

```diff
@@ -2,6 +2,26 @@ _ = require 'underscore'
 htmlparser = require 'htmlparser2'
 {makeSimpleStore} = require './helpers'
 
+LINKS_BEGIN = ['#']
+LINKS_CONTAIN = ['/contents/', '.cnx.org']
```

Comment:
JS parses the string `"http://cnx.org/contents/[UUID]"`, pull out `[UUID]`, and asks `/api/books/#/pages/[UUID]`